api/e2etests/e2e_seed_data.py

```diff
@@ -34,11 +35,41 @@ def seed_data() -> None:
     )
     org_admin.add_organisation(organisation, OrganisationRole.ADMIN)
 
-    project: Project = Project.objects.create(
-        name="My Test Project", organisation=organisation
-    )
-    Environment.objects.create(name="Development", project=project)
-    Environment.objects.create(name="Production", project=project)
+    project_test_data = [
+        {
+            "name": "My Test Project",
+            "environments": [
+                "Development",
+                "Production",
+            ],
+        },
+        {"name": "My Test Project 2", "environments": ["Development"]},
+        {"name": "My Test Project 3", "environments": ["Development"]},
+        {"name": "My Test Project 4", "environments": ["Development"]},
+    ]

```

Comment:
Can we add a comment explaining why different projects have different environments?

Reply:
Added.